stock_picking_invoicing/models/account_invoice.py

```diff
@@ -0,0 +1,66 @@
+# Copyright (C) 2019-Today: Odoo Community Association (OCA)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class AccountInvoice(models.Model):
+    _inherit = "account.invoice"
+
+    picking_ids = fields.Many2many(
```

Comment:
Can cause conflict with module  `stock_picking_invoice_link` https://github.com/OCA/stock-logistics-workflow/blob/12.0/stock_picking_invoice_link/models/account_invoice.py#L13

Add dependency to module stock_picking_invoice_link can be possible???